ui/src/components/SampleQueue/TodoTree.js

```diff
@@ -19,7 +19,6 @@ export default class TodoTree extends React.Component {
   showAddSampleForm() {
     this.props.sendPrepareForNewSample();
     this.props.showForm('AddSample');
-    this.props.showList('current');
```

Comment:
1. Go to the "Data Collection" page and select the "Queued Samples" tab
2. Click on "Create new sample"
3. Fill in form and click on "Add Sample" (now "Queue")
4. The UI switches to the mounted sample tab even though the new sample was only added to the queue, not mounted.

To fix this, `showList` is now called from the `addAndMount` function, in `AddSample`.